static/beta/stage/navigation/rhel-navigation.json

```
@@ -22,6 +22,20 @@
           "product": "Red Hat Insights",
           "description": "View details about your Red Hat Enterprise Linux systems."
         },
+        {
+          "id": "groups",
+          "appId": "groups",
```

I think we have to use the same appId for inventory

```suggestion
          "appId": "inventory",
```

---

Correct